src/core/dbus-execute.c

```diff
@@ -47,15 +47,13 @@
 #include "user-util.h"
 #include "utf8.h"
 
-BUS_DEFINE_PROPERTY_GET_ENUM(bus_property_get_exec_output, exec_output, ExecOutput);
-
-static BUS_DEFINE_PROPERTY_GET_ENUM(property_get_exec_input, exec_input, ExecInput);
-
```

Comment:
I've explicitly left the second iteration of this PR as a separate commit, as I was not happy with all the string-typing introduced with it. Your suggestion looks cleaner, I'll rework this PR in that sense.